cli/src/funTest/assets/semver4j-ort-result.yml

```diff
@@ -184,125 +184,140 @@ scanner:
     - "postgresStorage"
     storage_writers:
     - "postgresStorage"
+  provenances:
+  - id: "Maven:com.vdurmont:semver4j:3.1.0"
```

Comment:
The commit message should describe the actual change of the internal representation better, it currently focuses only on compatibility. I would suggest to add a first paragraph that shortly explains that scan results are now stored by provenance and that the package provenance resolution results are stored as well to associate scan results with packages.